stackit/internal/services/iaas/iaas_acc_test.go

```diff
@@ -83,6 +83,14 @@ var securityGroupResource = map[string]string{
 	"label1":      "value",
 }
 
+// Security Group rule resource data
+var securityGroupRuleResource = map[string]string{
+	"project_id":        testutil.ProjectId,
+	"security_group_id": securityGroupResource["security_group_id"],
```

Comment:
this can be removed right? I think it doesnt even exist :thinking:

Reply:
removed the obsolete field